docs/features/software-catalog/well-known-annotations.md

```diff
@@ -70,6 +70,25 @@ The value of this annotation is a location reference string (see above). If this
 annotation is specified, it is expected to point to a repository that the
 TechDocs system can read and generate docs from.
 
+### backstage.io/browser-view-url, backstage.io/browser-edit-url, backstage.io/browser-source-url
+
+```yaml
+# Example:
+metadata:
+  annotations:
+    backstage.io/browser-view-url: https://some.website/catalog-info.yaml
+    backstage.io/browser-edit-url: https://github.com/my-org/catalog/edit/master/my-service.jsonnet
+    backstage.io/browser-source-url: https://github.com/my-org/my-service
+```
+
+These annotations allow customising links from the catalog pages. The view URL
+should point to the canonical metadata YAML that governs this entity. The edit
+URL should point to the source file for the metadata. The source URL should
+point to the source code of the entity itself (where relevant, i.e. when the
+entity is a `Component`). In the example above, `my-org` generates its catalog
+data from Jsonnet files in a monorepo, and so for the `my-service` component, we
```

Comment:
Too outlandish for Vale, gotta add it to `.github/styles/vocab.txt` :grin:

Or tbh you could also make this `.jsonnet` since code blocks aren't checked

Reply:
Oh that's where it is. Cheers!